cfme/tests/openstack/cloud/test_networks.py

```diff
@@ -52,7 +52,7 @@ def create_subnet(appliance, provider, network):
 def create_router(appliance, provider, ext_gw, ext_network=None, ext_subnet=None):
     collection = appliance.collections.network_routers
     router = collection.create(name=fauxfactory.gen_alpha(),
-                               tenant=provider.data['tenant'],
+                               tenant=provider.data['provisioning']['cloud_tenant'],
```

Comment:
For good practice, I would use a .get() here instead of accessing the dict directly. That way the test won't throw a KeyError if that key does not exist. What do you think?